lib/oban/job.ex

```diff
@@ -47,13 +47,29 @@ defmodule Oban.Job do
           | :worker
         ]
 
+  @type schedule_in_option ::
+          pos_integer()
+          | {pos_integer(),
+             :second
+             | :seconds
+             | :minute
+             | :minutes
+             | :hour
+             | :hours
+             | :day
+             | :days
+             | :week
+             | :weeks
+             | :month
+             | :months}
```

Comment:
Due to the variable number of days in a month and the expected accuracy of `scheduled_at,` I don't think we should support months. Of course, weeks may differ between calendars as well, but as this is just syntax sugar, I think using weeks is fine.